app/models/patient.rb

```diff
@@ -215,11 +216,19 @@ def search(name_or_phone_str, lines = LINES)
       all_matching_names = find_name_matches name_regexp, lines
       all_matching_phones = find_phone_matches phone_regexp, lines
 
-      (all_matching_names | all_matching_phones)
+      sort_and_limit_patient_matches(all_matching_names, all_matching_phones)
```

Comment:
You may want to pull from master -- @Kevin-Wei added a `search by identifier` to this concatenation.